css/style.css

```diff
@@ -1025,4 +2392,102 @@
   .arrow {
     padding-left: 2px;
   }
+
+  .wrap-information,
+  .wrap {
+    display: flex;
+    justify-content: flex-end;
+    flex-direction: column;
+    background-position-y: top;
+    height: 510px;
+  }
+
+  .info__title {
+    display: none;
+  }
+
+  .info__img {
+    height: auto;
+    width: 100%;
+    max-width: 374px;
+  }
+
+  .red-first,
+  .details {
+    max-width: 100%;
+    height: 300px;
+  }
+
+  .red-first {
+    padding: 20px 20px 0 20px;
+  }
+
+  .description__title,
+  .description__title-most {
+    font-size: 24px;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5839070)